SalesforceCommerceCloud/pwa-kit
# [Home page] Add Shop Products section with Products from the Catalog

GUS: https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000Q5jCCYAZ/view
# Description
This PR adds the Shop Products scroller to the Home page with Products from the first category on the catalog.

The products are obtained server-side using the `product-search` endpoint via the`getProps` API.



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Added Shop Products section using products from the `newarrivals` category.

# How to Test-Drive This PR

- Check out this branch `category-products`.
- Run `npm run start --prefix packages/pwa`
- Open the Network tab in Chrome Dev Tools.
- Load the Home page http://localhost:3000/
- Verify that the Shop Product section uses products from the first category in the catalog by sending a request to the `product-search` endpoint.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
